app/frontend/components/shared/ui/shared-spinner.tsx

```diff
@@ -0,0 +1,8 @@
+import { Spinner, SpinnerProps } from "@chakra-ui/react"
+import React from "react"
+
+interface ISharedSpinner extends SpinnerProps {}
+
+export const SharedSpinner = ({ ...rest }: ISharedSpinner) => {
+  return <Spinner {...rest} />
+}
```

Comment:
this seems quite unnecessary as it's just wrapping the chakra Spinner?

Reply:
This is a stub for when we want a site wide standardized spinner later, let's leave it for now?